vernac/vernacstate.ml

```diff
@@ -93,19 +93,16 @@ type t = {
   parsing : Parser.t;
   system  : System.t;              (* summary + libstack *)
   lemmas  : LemmaStack.t option;   (* proofs of lemmas currently opened *)
-  program : Declare.OblState.t NeList.t;    (* obligations table *)
   opaques : Opaques.Summary.t;     (* opaque proof terms *)
```

Comment:
I need to think about this, as in a sense these two are isomorphic, you are now just embedding the state into `System.t`.